docs/terminology.md

```diff
@@ -4,35 +4,49 @@ On this page you can quickly get a grasp of common UiTPAS terminology and concep
 
 ## Passholder
 
-One who holds an UiTPAS, the end-user of an UiTPAS.
+One who holds an UiTPAS, the end-user of an UiTPAS, identified by an UiTPAS number.
+
+## UiTPAS Event
+
+Certain actions within UiTPAS are linked to specific events, for example when[registering a ticket sale](./registering-ticket-sales.md) at a discounted UiTPAS tariff.
+
+Those events need to exist in [UiTdatabank](https://www.uitdatabank.be/). UiTdatabank is a central database of cultural and leisure activities in Flanders and Brussels. [To register an UiTPAS event](./registering-events.md), the event in UiTdatabank needs to have a base price and has to be linked to a known UiTPAS organizer.
 
 ## UiTPAS Organizer
 
 The organizer of an UiTPAS event. To pay for the social tariff for people in need, the organizer is partly reimbursed by the local government. That's why it's important to always create UiTPAS events with the correct UiTPAS organizer.
```

Comment:
Niet per se nodig in deze PR, maar we zouden misschien ook best uitleggen dat organizers ook in UiTdatabank worden opgeslagen zoals je hierboven hebt gedaan voor events, omdat we daar alle info bijhouden rond activiteiten, de locaties waar ze plaatsvinden, en wie ze organiseert (los van UiTPAS).
Dat was niet zo duidelijk voor de dev van Intracto die nu een integratie bouwt voor de scholen app.

Reply:
heb het er toch al ingestoken!! Is idd niet vanzelfsprekend - evt ook nog in de Guide 'registering events'

Reply:
De link die je nu gebruikt hebt is wel niet bruikbaar voor mensen buiten publiq. Die is enkel voor admins.
Ik zou voorlopig precies gewoon linken naar https://publiq.stoplight.io/docs/uitdatabank/ en als we daar dan de documentatie aangevuld hebben dan kunnen we de link hier nog aanpassen naar een die specifiek naar de documentatie over organisatoren verwijst.